jolt-core/src/jolt/instruction/sw.rs

```diff
@@ -65,24 +71,59 @@ mod test {
     use crate::{jolt::instruction::JoltInstruction, jolt_instruction_test};
 
     #[test]
-    fn sw_instruction_e2e() {
+    fn sw_instruction_32_e2e() {
         let mut rng = test_rng();
-        const C: usize = 4;
+        const C: usize = 2;
```

Comment:
Can we keep C = 4 for the 32-bit test and C = 8 for the 64-bit test? Those are probably the parameters we'd use in practice

Reply:
Yes, that is totally fine. I think you're just wasting lookups though, since for 16-bit chunk length, you only need 2 chunks for 32-bit and 4 chunks for 64-bit.